### ISSUE TITLE
dataset download not working

### ISSUE THREAD

**Issue by nikit-srivastava** [CoDS-GCS/KGQAn#7]
Hello,

Thank you for the nice work.

I would like to benchmark your system for my research work, however, I have been facing an error while running the [data_download.sh](https://github.com/CoDS-GCS/KGQAn/blob/dev/data_download.sh) script.

In your data_download.sh script there seems to be a problem while executing [line 21](https://github.com/CoDS-GCS/KGQAn/blob/1c10a596fde8aeefe19a7565c4e572bb18b1b06a/data_download.sh#L21) and [line 26](https://github.com/CoDS-GCS/KGQAn/blob/1c10a596fde8aeefe19a7565c4e572bb18b1b06a/data_download.sh#L21). It seems that the zip files being downloaded in these lines are corrupted. Looking further into this issue, I found that the wget statement is not downloading the zip files correctly. Moreover, the google drive [link](https://docs.google.com/uc?export=download&id=149tB7x_dYLfmwoEdCfKfjBNeGdqFvqOP) at [line 26](https://github.com/CoDS-GCS/KGQAn/blob/1c10a596fde8aeefe19a7565c4e572bb18b1b06a/data_download.sh#L21) does not work. To fix the `wget` issue, I would like to suggest the following at the line 21:
`curl -L "https://drive.usercontent.google.com/download?id=1QbT5FDOJtdVd7AqZ-ekwUh2_pn6nNpb3&confirm=xxx" -o output_pred21_8_30.zip`

For, `wiki-news-300d-1M.zip` you probably need to check the permissions of the zip file on your drive or reupload it.

**Comment by Reham-Osama** [comment]
Hello, 

Can you please try to download the files from these two links and let me know if they work:
1. wiki-news-300d-1M.zip: https://drive.google.com/file/d/1UTPGv8QUgqSVQ2JeX9QVW0YhbGRxONLL/view?usp=sharing
2. output_pred21_8_30.zip: https://drive.google.com/file/d/1QbT5FDOJtdVd7AqZ-ekwUh2_pn6nNpb3/view?usp=sharing

It looks like there was an update in Google Drive that hinders downloading big files in the way used in the script. 
For now, you can comment Line 21 and 26 and download the files using these provided links.

**Comment by nikit-srivastava** [comment]
@Reham-Osama Thank you for making the links available, the dataset download now seems to complete normally. However, I encountered an error where the word_embedding docker container was unable to find the embedding file. I managed to resolve it by changing [line 8 @ docker-compose-server.yml](https://github.com/CoDS-GCS/KGQAn/blob/1c10a596fde8aeefe19a7565c4e572bb18b1b06a/docker-compose-server.yml#L8) to `- ./data/wiki-news-300d-1M.txt:/app/data/wiki-news-300d-1M.txt`.
